src/test_utils/non_regression.rs

```diff
@@ -178,7 +172,7 @@ impl NonRegressionTestVector {
         let reg_vectors = Self {
             public_key: transcoder.encode(mpk.serialize()?),
             master_secret_key: transcoder.encode(msk.serialize()?),
-            policy: transcoder.encode(<Vec<u8>>::try_from(&policy).unwrap()),
+            policy: policy.to_string(),
```

Comment:
Remove the policy field instead (no need to serialize it in the regression vector since it is already present in the serialized MPK):
```suggestion
```